ironfish/src/network/peerNetwork.test.ts

```diff
@@ -867,7 +882,14 @@ describe('PeerNetwork', () => {
 
         const accountA = await useAccountFixture(wallet, 'accountA')
         const accountB = await useAccountFixture(wallet, 'accountB')
-        const { block, transaction } = await useBlockWithTx(node, accountA, accountB)
+        const { block, transaction } = await useBlockWithTx(
```

Comment:
Updating fixtures here. This test should not sync to mempool when creating this transaction